ui-spacetimechart/src/lib/consts.ts

```diff
@@ -6,6 +6,7 @@ export const BLUE = '#2170B9';
 export const GREY_10 = '#EDEDED';
 export const GREY_30 = '#B6B2AF';
```

Comment:
```suggestion
export const GREY_30 = '#B6B3AF';
```

seems it has been changed in the mockup